tests/test_remesh.py

```diff
@@ -144,22 +180,25 @@ def fixture_gyroid_mesh() -> BoxMesh:
 def test_given_periodic_mesh_remesh_keeping_periodicity_for_fem_must_maintain_periodicity(
     shape: BoxMesh,
     request,

```

Comment:
Do we need this `request` argument for this test ? Also the type hint is missing

Reply:
Since the fixture `shape` is actually considered a `str` (my bad, it has been corrected now) because of parametrize, `request` is needed to fetch its actual value and type. I added its type hint as well.